core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_rh_appstudio

```diff
@@ -0,0 +1 @@
+quay.io/redhat-appstudio-qe/e2e-test-runner:latest registry.ci.openshift.org/rh-appstudio/e2e-test-runner:latest
```

Comment:
```suggestion
quay.io/redhat-appstudio-qe/e2e-test-runner:latest registry.ci.openshift.org/ci/e2e-test-runner:latest
```
And move the line to the existing file?

I guess the source image is already open to the public.
It would be easier to use an existing ns.